connector_dns_dnspod/dnspod.py

```diff
@@ -0,0 +1,159 @@
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, api
+from .backend import dnspod
+from .unit.export_synchronizer import DNSExporter
+from .unit.backend_adapter import DNSPodAdapter
+from openerp.addons.connector.unit.mapper import (
+    mapping, ExportMapper)
+import httplib
+import urllib
+import json
+
+
+class DNSPodBackend(models.Model):
+    _inherit = 'dns.backend'
+
+    def _select_version(self):
+        """return version selection"""
+        res = []
+        res.append(('dnspod', 'dnspod'))
+        return res
+
+    @api.multi
+    def params(self):
+        return {'format': 'json', 'login_email': self.login,
+                'login_password': self.password}
+
+    def request(self, action, params, method='POST'):
+        """send request to 'dnsapi.cn'"""
+        headers = {
+            "Content-type": "application/x-www-form-urlencoded",
+            "Accept": "text/json"
+        }
+        conn = httplib.HTTPSConnection("dnsapi.cn")
+        conn.request(method, '/' + action, urllib.urlencode(params), headers)
+        response = conn.getresponse()
+        data = response.read()
+        conn.close()
+        if response.status == 200:
+            return data
+        else:
+            return None
+
+    @api.one
+    def button_connect(self):
+        params = self.params()
+        data = self.request('Domain.List', params)
+        data = json.loads(data)
+        if int(data['status']['code']) != -1:
+            self.state = 'done'
+        else:
+            self.state = 'exception'
+
+    @api.one
+    def button_set_draft(self):
+        self.state = 'draft'
+
+
+class DNSPodDomain(models.Model):
+    _inherit = 'dns.domain'
+
+
+class DNSPodRecord(models.Model):
+    _inherit = 'dns.record'
+
+    def _line_select_version(self):
+        """return record line selection"""
+        res = [('A', '默认'), ('B', '电信'), ('C', '联通'),
+               ('D', '教育网'), ('E', '百度'), ('F', '搜索引擎')]
+        return res
+
+    def _type_select_version(self):
+        """return record type selection"""
+        res = [('A', 'A'), ('CNAME', 'CNAME'), ('MX', 'MX'),
+               ('TXT', 'TXT'), ('NS', 'NS'), ('AAAA', 'AAAA'),
+               ('SRV', 'SRV'), ('Visibile URL', '显性URL'),
```

Comment:
s/Visibile/Visible/

Same remark as above: keep the strings in the code in english and move the chinese version in the translation file.